[EDU-4818] feat: Update home texts and cards (#1184)
aziontech/docs: src/content/docs/en/homes/home-docs.mdx

      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: Bot Management
        description: >-
          Use Azion Bot Manager to protect edge applications from bots,
          preventing malicious attacks.
        link: >-
          /en/documentation/architecture/bot-management/protect-your-applications-with-bot-management/
      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: WAAP
        description: >-
          Secure applications and APIs against cyber threats from OWASP Top 10,
          zero-day attacks, and bot activities.
        link: >-
          /en/documentation/architecture/edge-firewall/web-application-and-api-protection-waap/
      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: Serverless Applications
        description: >-
          Explore Azion Edge Functions for serverless applications, enhancing
          scalability, performance, and real-time responsiveness at the edge.
        link: >-
          /en/documentation/architectures/edge-application/serverless-applications/
      - icon: /assets/docs/images/uploads/icon-guides.svg

>         title: Find more ...

Reviewer: ```suggestion
        title: Find more
```

        description: >-

>           View all use cases

Reviewer: ```suggestion
          View all use cases.
```

        link: >-
          /en/documentation/architectures/
  - title: Pick your development interface
    cards:
      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: Azion Console
        description: >- 

>           Manage and monitor your edgeapplications, services, and resources efficiently.

Reviewer 2: ```suggestion
          Manage and monitor your edge applications, services, and resources efficiently.
```

meta_tag_robots_no_index: false
title: Welcome to Azion Docs
description: >-
  Azion is the full-stack edge computing platform that simplifies how you build, secure, and scale modern applications.
meta_tags: 'documentation, azion'
namespace: documentation_home
product_cards:
  - title: 'Build applications faster'
    description: >-
       Build using your favorite framework and deploy an edge application in just a few steps.
    cards:
      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: React
        description: 'Build reusable UI components. Create dynamic web apps for edge infrastructure deployment.'
        link: 'https://console.azion.com/create/react/react-boilerplate'
      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: Next

>         description: 'Create React apps with automatic routing and optimization for edge deployment.'

Reviewer: ```suggestion
        description: 'Create Next apps with automatic routing and optimization for edge deployment.'
```

        link: 'https://console.azion.com/create/nextjs/nextjs--static-boilerplate'
      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: Vue
        description: 'Create progressive, scalable interfaces. Deploy modern Vue.js applications to the edge.'
        link: 'https://console.azion.com/create/vue/vue-boilerplate'
      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: Angular
        description: 'Develop robust, scalable web applications using TypeScript for rapid edge deployment.'
        link: 'https://console.azion.com/create/angular/angular-boilerplate'
      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: Svelte
        description: 'Develop reactive UIs with less code, compiled for high-performance edge deployment.'
        link: 'https://console.azion.com/create/svelte/svelte-boilerplate'  
      - icon: /assets/docs/images/uploads/icon-guides.svg
        title: Gatsby

>         description: 'Build fast, SEO-optimized sites using React and GraphQL for efficient edge delivery.'

Reviewer: ```suggestion
        description: 'Build fast, SEO-optimized sites using Gatsby for efficient edge delivery.'
```

        link: /en/documentation/products/terraform-provider/
  - title: 'Choose your journey'
    cards:
      - icon: /assets/docs/images/uploads/icon-first-setps.svg
        title: Get Started
        description: Initial guide to moving your applications to Azion.
        link: /en/documentation/products/get-started/
      - icon: /assets/docs/images/uploads/edge-application.svg
        title: Build
        description: >-
          Create with powerful tools and frameworks.
        link: /en/documentation/products/guides/build/overview/
      - icon: /assets/docs/images/uploads/my-domains.svg
        title: Secure
        description: >-
          Fully protect your applications.
        link: /en/documentation/products/secure/overview/
      - icon: /assets/docs/images/uploads/edge-node.svg
        title: Deploy
        description: >-
          Manage edge resources efficiently.
        link: /en/documentation/products/deploy/overview/
      - icon: /assets/docs/images/uploads/data-stream.svg
        title: Observe
        description: >-
          Monitor and gain analytics in real-time.
        link: /en/documentation/products/observe/overview/

>   - title: 'What is new?'

Reviewer: ```suggestion
  - title: 'What's new?'
```
